android/app/src/main/java/com/msupplymobile/MainActivity.java

```diff
@@ -1,18 +1,8 @@
 package com.msupplymobile;
 
 import com.facebook.react.ReactActivity;
-import com.babisoft.ReactNativeLocalization.ReactNativeLocalizationPackage;
-import io.realm.react.RealmReactPackage;
```

Comment:
I think you need to relink native modules. `react-native link` or something along those lines should do it. Unless it's simply all removed from here and only in `MainApplication.java` now.

Reply:
I did link the modules, which was supposed to add these lines to the MainActivity.java but it didn't. Both realm and localization are working without these lines so I did not manually add them.